src/assets/img/icons/enter_private_key.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M10.7757 17.2243C10.5414 16.99 10.5414 16.6101 10.7757 16.3758L13.2686 13.8829C13.4248 13.7267 13.4248 13.4734 13.2686 13.3172L10.7757 10.8243C10.5414 10.59 10.5414 10.2101 10.7757 9.97579C11.0101 9.74147 11.3899 9.74147 11.6243 9.97579L14.1172 12.4687C14.2734 12.6249 14.5266 12.6249 14.6828 12.4687L17.1757 9.97579C17.4101 9.74147 17.7899 9.74147 18.0243 9.97579C18.2586 10.2101 18.2586 10.59 18.0243 10.8243L15.5314 13.3172C15.3752 13.4734 15.3752 13.7267 15.5314 13.8829L18.0243 16.3758C18.2586 16.6101 18.2586 16.99 18.0243 17.2243C17.79 17.4586 17.4101 17.4586 17.1757 17.2243L14.6828 14.7314C14.5266 14.5752 14.2734 14.5752 14.1172 14.7314L11.6243 17.2243C11.3899 17.4586 11.0101 17.4586 10.7757 17.2243Z" fill="#4360DF"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M5.2 2.80005C3.43269 2.80005 2 4.23274 2 6.00005V14.0001C2 15.7674 3.43269 17.2001 5.2 17.2001H6C7.76731 17.2001 9.2 15.7674 9.2 14.0001V6.00005C9.2 4.23274 7.76731 2.80005 6 2.80005H5.2ZM6 4.00005H5.2C4.09543 4.00005 3.2 4.89548 3.2 6.00005V11.0687C3.2 11.2469 3.41543 11.3361 3.54142 11.2101L7.97574 6.77579C7.99077 6.76075 8 6.74066 8 6.7194V6.00005C8 4.89548 7.10457 4.00005 6 4.00005ZM3.2 13.2807C3.2 13.2594 3.20923 13.2394 3.22426 13.2243L7.65858 8.79C7.78457 8.66401 8 8.75324 8 8.93142V14.0001C8 15.1046 7.10457 16.0001 6 16.0001H5.2C4.09543 16.0001 3.2 15.1046 3.2 14.0001V13.2807Z" fill="#4360DF"/>
+</svg>
```

Comment:
All icons living in `img/icons` need to have a 24x24 container around it. Can you update this asset accordingly please?

Reply:
done